### DISCUSSION TITLE
s/QG42UqJ9rnswQDMd41CYUg

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#44] [Announcements]
# s/QG42UqJ9rnswQDMd41CYUg

Kerrigan：配置中心管理UI的实现思路和技术细节

https://blog.ops-coffee.cn/s/QG42UqJ9rnswQDMd41CYUg

**Comment by yc550370460** [comment]
有点疑惑，confd貌似不能实现配置文件的新增和删除吧？但你的WEB ui有新增和删除功能， 没理解到。

**Reply by ops-coffee** [reply]
confd的数据来自于etcd，所以只需要对etcd做修改，confd会同步，webui数据会入库，在合适的时候去修改etcd里的数据

**Reply by yc550370460** [reply]
我的意思是， 比如使用confd + etcd去监控nginx配置文件， conf目录下面， 为了方便管理， 一般是每个域名一个conf文件， 如果我想要新增一个域名， 或者一个域名想要下线需要删掉对应的conf文件， 这种场景是不是不能实现？而只能将所有域名放到一个conf文件下面， 提前配置好confd的dest来实现？

**Reply by ops-coffee** [reply]
就像你想的那样，对于你的这个场景，我们有另一个轮子来实现 https://blog.ops-coffee.cn/s/FV379mGIboVYky73kcoMMQ

**Reply by yc550370460** [reply]
了解啦， 感谢老哥， 看看新轮子
